src/components/standalone/users_database/UsersTable.vue

```diff
@@ -28,6 +30,8 @@ const props = defineProps<{
 const emit = defineEmits<{
   delete: [item: User]
   edit: [item: User]
+  set_admin: [item: User]
+  remove_admin: [item: User]
```

Comment:
Consider following this format for events (generally for everything in Vue): https://vuejs.org/guide/components/events.html#emitting-and-listening-to-events